drivers/SmartThings/matter-switch/src/test/test_matter_multi_button.lua

```diff
@@ -0,0 +1,732 @@
+local test = require "integration_test"
+local capabilities = require "st.capabilities"
+local t_utils = require "integration_test.utils"
+local utils = require "st.utils"
+local dkjson = require "dkjson"
+
+local clusters = require "st.matter.generated.zap_clusters"
+local button_attr = capabilities.button.button
+
+-- Mock a 4-button device using endpoints non-consecutive endpoints
```

Comment:
nit: a mock 5-button device

Reply:
Fixed in 340df0f5c6c6d95d4d9b7c19e2b70b4b2fb646e2